packages/server/test/question.integration.ts

```diff
@@ -39,7 +53,8 @@ describe('Question Integration', () => {
 
   describe('POST /questions', () => {
     it('posts a new question', async () => {
-      const queue = await QueueFactory.create();
+      const course = await CourseFactory.create();
+      const queue = await QueueFactory.create({ courseId: course.id });
```

Comment:
Just curious why we need this change?

Reply:
The problem with the guard is that then you need to explicitly have a queue and course for it to run successfully which is true when you're running the app, but not in this test.